src/anemoi/graphs/create.py

```diff
@@ -57,6 +57,7 @@ def update_graph(self, graph: HeteroData) -> HeteroData:
 
         for edges_cfg in self.config.get("edges", {}):
 
+            # Remove in a future version
             if "edge_builder" in edges_cfg:
```

Comment:
shouldn't this logic be in the `__init__` ?

Reply:
Moved to `__init__`.